dataset/dictionary/enemies.json5

```
@@ -695,29 +810,40 @@
   {
     en: "Electro Whopperflower",
     ja: "トリックフラワー・雷",
+    zh: "电气骗骗花",
     pronunciationJa: "トリックフラワー・かみなり",
     variants: {
       en: [ "Electro Trickflower", "Electro Trick Flower" ],
+      zh: [ "雷骗骗花" ],
     },
     tags: [ "enemy" ],
   },
   {
     en: "Pyro Regisvine",
     ja: "爆炎樹",
+    zh: "爆炎树",
     pronunciationJa: "ばくえんじゅ",
+    variants: {
+      zh: [ "火树" ],
+    }
```

Comma seems missing.
You can find syntax errors by running `npm run lint`.

```suggestion
    },
```

---

Sorry for it and thanks for your advice